src/easier68k/core/util/parsing.py

```diff
@@ -24,21 +122,22 @@ def parse_literal(literal: str):
         if len(literal) % 2 == 0:
             literal = literal[0] + '0' + literal[1:]
 
-        return bytearray.fromhex(literal[1:])
+        return int(literal[1:], 16)
```

Comment:
Does this mean our goal is to pass ints everywhere? We should probably make sure to do a check through the project to make it consistent.